waspc/src/Wasp/AppSpec/PackageJson.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module Wasp.AppSpec.PackageJson where
+
+import Control.Applicative (liftA2)
+import Data.Aeson.TH
+import Data.Map (Map)
+import qualified Data.Map as M
+import GHC.Generics (Generic)
+import Wasp.AppSpec.App.Dependency (Dependency)
+import qualified Wasp.AppSpec.App.Dependency as D
+
+data PackageJson = PackageJson
+  { _name :: !String,
+    -- todo(filip): do this properly once you merge martin's PR
+    _dependencies :: !(Map String String),
+    _devDependencies :: !(Map String String)
+  }
+  deriving (Show, Generic)
+
+$(deriveJSON defaultOptions {fieldLabelModifier = drop 1} ''PackageJson)
+
+dependencies :: PackageJson -> [Dependency]
+dependencies packageJson = D.fromList $ M.toList $ _dependencies packageJson
+
+devDependencies :: PackageJson -> [Dependency]
+devDependencies packageJson = D.fromList $ M.toList $ _devDependencies packageJson
+
+allDependencies :: PackageJson -> [Dependency]
```

Comment:
Why would somebody want to import all deps and lose info about which ones are dev and which ones are non-dev?

Reply:
I added this to make it backwards compatible with our current system that doesn't support dev deps (i.e., I added a layer that ignores the distinction on purpose).

However, I have since fully abandoned the legacy system of not differentiating between dependencies, so I'll remove this function.

Reply:
Got it! Feel free then to resolve this comment once you don't need it any more.